packages/tokens/non-fungible/src/extensions/consecutive/storage.rs

```diff
@@ -0,0 +1,341 @@
+use soroban_sdk::{contracttype, panic_with_error, Address, Env};
+
+use super::emit_consecutive_mint;
+use crate::{
+    burnable::emit_burn,
+    emit_transfer,
+    sequential::{self as sequential},
+    storage::{approve_for_owner, check_spender_approval, decrease_balance, increase_balance},
+    NonFungibleTokenError, TokenId,
+};
+
+/// Storage keys for the data associated with `FungibleToken`
+#[contracttype]
+pub enum StorageKey {
+    Approval(TokenId),
+    Owner(TokenId),
+    BurnedToken(TokenId),
+}
+
+// ################## QUERY STATE ##################
+
+/// Returns the address of the owner of the given `token_id`.
+///
+/// # Arguments
+///
+/// * `e` - Access to the Soroban environment.
+/// * `token_id` - Token id as a number.
+///
+/// # Errors
+///
+/// * [`NonFungibleTokenError::NonExistentToken`] - Occurs if the provided
+///   `token_id` does not exist.
+pub fn owner_of(e: &Env, token_id: TokenId) -> Address {
+    let max = sequential::next_token_id(e);
+    let is_burned =
+        e.storage().persistent().get(&StorageKey::BurnedToken(token_id)).unwrap_or(false);
+
+    if token_id >= max || is_burned {
+        panic_with_error!(&e, NonFungibleTokenError::NonExistentToken);
+    }
+
+    //e.storage().persistent().extend_ttl(&key, OWNER_TTL_THRESHOLD,
```

Comment:
commented out line

Reply:
it will be better handled with all the others in https://github.com/OpenZeppelin/stellar-contracts/issues/129